### DISCUSSION TITLE
Is there a way to change the default browser only for the Stability Matrix app

### DISCUSSION THREAD

**Discussion by saadkhan955** [LykosAI/StabilityMatrix#503] [Q&A]
I want to associate another browser as the default specifically for the Stability Matrix app. For example, I have Edge as my overall default browser, but I want to use Mozilla Firefox as the default browser when opening links or clicking on the launch button from Stability Matrix.

**Comment by FullBleed** [comment]
How big of a problem is this?  I don't want to use my default browser with Stability Matrix.

**Comment by schoenid** [comment]
Same to me.
I'm mainly working on Opera, but would like to open Stability UIs on Firefox. 
This because Opera has often memory problems on Stable Diffusion.

**Comment by bkrew** [comment]
It would be a nice feature, sometimes updates on a browser breaks some functions. 
So the Option to choose a browser would be appreciated.

**Comment by schoenid** [comment]
The browser choice should be an option in the settings for every installad package. 
This would allow to define specific browsers for every package. 

If not selected, the default browser should be used.

If the option is selected, there could be either a list of installed browsers, or at least a field to specify the path to the desired browser. Maybe both could be available.

**Comment by XDM-Inc** [comment]
i too would like to have this open on a separate browser. for the matter of keeping my AI work separate from my web browser (play). also  Stability Matrix leaves things like history and file names which could be a security vulnerability. the work around for now is to just copy the address and open in whatever browser or private mode.
